README.md

```diff
@@ -44,10 +44,28 @@ cd 01/software # CD into the source directory
 ```shell
 brew install portaudio ffmpeg cmake # Install Mac OSX dependencies
 poetry install # Install Python dependencies
-export OPENAI_API_KEY=sk... # OR run `poetry run 01 --local` to run everything locally
+export OPENAI_API_KEY=sk... 
 poetry run 01 # Runs the 01 Light simulator (hold your spacebar, speak, release)
 ```
 
+### Running locally
+To run locally, you can use command line arguments and environment variables.
+
+Using command line arguments:
+
+```shell
+poetry run 01 --local --model ollama/mixtral:latest
+```
+
+Using command line arguments and environment variables:
+
+```shell
+export MODEL=ollama/mixtral:latest
+poetry run 01 --local
+```
+
+Note, you should replace `ollama/mixtral:latest` with a model installed locally. For supported models, see Open Interpreter's [local providers](https://docs.openinterpreter.com/language-models/local-models/lm-studio) documentation.
```

Comment:
I wonder if this documentation should point to either ollama's local providers docs within Open Interpreter:

https://docs.openinterpreter.com/language-models/local-models/ollama

Or the ollama library directly:

https://ollama.com/library

As it is, linking to the LM Studio documentation within Open Interpreter isn't entirely clear.

Reply:
It's a good point. Ideally there would be a [local-models](https://docs.openinterpreter.com/language-models/local-models) parent article in Open Interpreter's documentation. But it defaults to [lm-studio](https://docs.openinterpreter.com/language-models/local-models/lm-studio) at the moment.

Since I use `ollama/mixtral:latest` as an example, it's clearer if I reference Open Interpreter's [ollama](https://docs.openinterpreter.com/language-models/local-models/ollama) article. I'll make the change.